models/projects.js

```diff
@@ -37,15 +37,17 @@ module.exports = {
         syncedAt : Date,
         isDeleted : {
             type : Boolean,
-            default : false
+            default : false,
+            index: true
         },
         categories : {
             type : Array,
             default : []
         },
         createdBy : {
             type : String,
-            default : "SYSTEM"
+            default : "SYSTEM",
```

Comment:
@priyanka-TL as discussed please remove this index and also the query from code.